### ISSUE TITLE
Add check for param type indention in PEAR.Commenting.FunctionCommentSniff.

### ISSUE THREAD

**Issue by Konafets** [squizlabs/PHP_CodeSniffer#418]
The sniff checks for proper alignment of doc comment params, but doesn't take the indention of the param type into account.  

```php
    /**
     * Param not immediate
     *
     * @param   int   $threeSpaces The indention of int is wrong but doesn't raise an error
     * @param int     $superfluous
     * @param missing
     * @param
     */
```

This code is taken from here https://github.com/squizlabs/PHP_CodeSniffer/blob/master/CodeSniffer/Standards/PEAR/Tests/Commenting/FunctionCommentUnitTest.inc#L123

**Comment by gsherwood** [comment]
What @aik099 said. The grouping of tags and the spacing after a tag name is a generic check for all block comments. I don't replicate it in all function/class/var comment sniffs and instead just include the DocComment sniff to do those generic checks.
